src/test/java/core/basesyntax/service/impl/FruitStrategyImplTest.java

```diff
@@ -0,0 +1,56 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.service.FruitStrategy;
+import core.basesyntax.strategy.BalanceOperation;
+import core.basesyntax.strategy.FruitOperation;
+import core.basesyntax.strategy.PurchaseOperation;
+import core.basesyntax.strategy.ReturnOperation;
+import core.basesyntax.strategy.SupplyOperation;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FruitStrategyImplTest {
+    private static FruitStrategy strategy;
+    private FruitOperation expectedOperation;
+    private FruitOperation actualOperation;
+
+    @BeforeClass
+    public static void beforeClass() {
+        strategy = new FruitStrategyImpl();
+    }
+
+    @Test(expected = RuntimeException.class)
+    public void makeOperation_WithWrongAbbreviation_notOk() {
+        strategy.makeOperation("d");
+    }
+
+    @Test
+    public void makeOperation_BalanceOperation_Ok() {

```

Comment:
```suggestion
    public void makeOperation_balanceOperation_ok() {
```
the first letter of each part of the method name should be lowercase